installer/roles/image_build/files/Dockerfile.sdist

```diff
@@ -1,20 +1,17 @@
-FROM centos:7
+FROM centos:8
 
-RUN yum -y update && yum -y install epel-release && yum -y install https://centos7.iuscommunity.org/ius-release.rpm
```

Comment:
Nice catch, now that we are using the rhel8 container, we definitely don't need this any more.  PG10 is in RHEL8.